projects/components/src/breadcrumbs/breadcrumbs.component.test.ts

```diff
@@ -53,12 +57,20 @@ describe('BreadcrumbsComponent', () => {
     expect(spectator.queryAll('.breadcrumbs .breadcrumb').length).toBe(3);
   });
 
-  it('should show icon only on the first crumb', () => {
+  it('should show icon on the first crumb', () => {
     const crumbs = spectator.queryAll('.breadcrumbs .breadcrumb');
     expect(crumbs[0].querySelector('ht-icon')).toExist();
+  });
 
-    crumbs.shift();
-    crumbs.forEach(crumb => expect(crumb.querySelector('ht-icon')).toBeNull());
+  it('should display the icon', () => {
+    const crumbs = spectator.queryAll('.breadcrumbs .breadcrumb');
+    crumbs.forEach((crumb, index) => {
+      if (index < crumbs.length - 1) {
```

Comment:
We'd expect icon to be null only at index 2 (line 46 above), but at index 0 (due to the `isFirst`) and index 1 (due to alwaysShowIcon on line 39)